libra-node/src/main_node.rs

```diff
@@ -62,8 +63,9 @@ fn setup_debug_interface(config: &NodeConfig) -> NodeDebugService {
 
 // TODO(abhayb): Move to network crate (similar to consensus).
 pub fn setup_network(
```

Comment:
Could we add a info! level logging message with the chain-id?  This could be helpful in debugging situations where some nodes are on different chain-ids and all we have are the logs.

Reply:
Actually, scratch this, looks like all config is already printed.